### PR TITLE
Update icons and add "deprecated" status

### PR DESCRIPTION
**Pull request by nightkin**
subj

### CODE REVIEW

File: assets/css/specStatus.css [hunk 1]
```diff
@@ -31,49 +31,56 @@
...
 @media only screen and (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (-webkit-min-device-pixel-ratio: 1.5), (min-device-pixel-ratio: 1.5) {
   .source_status_badge {
-    top: 5px !important;
-    background-image: url(../i/dev.png) !important;
+    background-image: url(../i/development_s@2x.png);
   }
 }
 .source_status_badge.__exp {
-  background-image: url(../i/exp__sm.png)  !important;
+  background-image: url(../i/experiment_s.png);
```
Comment (operatino) [inline assets/css/specStatus.css]: Did you tested that it's working without important? Most likely not

**Comment by operatino** [comment]
And please add small description how to add statuses on sections.

**Comment by operatino** [comment]
Thank you for updates, I published new version with them to NPM.

**Comment by nightkin** [comment]
Thanks!

**Pull request merged**